src/structures/GuildAuditLogs.js

```diff
@@ -281,7 +281,8 @@ class GuildAuditLogsEntry {
     } else if ([Targets.USER, Targets.GUILD].includes(targetType)) {
       this.target = guild.client[`${targetType.toLowerCase()}s`].get(data.target_id);
     } else if (targetType === Targets.WEBHOOK) {
-      this.target = this.webhooks.get(data.target_id);
+      this.target = logs.webhooks.get(data.target_id) ||
+        new Webhook(guild.client, { id: data.target_id, guild_id: guild.id });
```

Comment:
The `changes` array of the data has `channel_id` and `name` of the webhook, those could also be passed to the constructor.